src/Stripe.net/Services/PaymentIntents/PaymentIntentConfirmOptions.cs

```diff
@@ -0,0 +1,26 @@
+namespace Stripe
+{
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+
+    public class PaymentIntentConfirmOptions : StripeBaseOptions
+    {
```

Comment:
According to the OpenAPI spec, this request also has an additional `client_secret` string parameter.

Reply:
That's a client-side property used with the publishable key, I'm not sure this is a useful parameter in the server-side library. Do you still think I should add it? It's not in our docs.

Reply:
Ack, let's ignore it for now -- we can always append it later if needed.